### DISCUSSION TITLE
UTF-8 but no Diacritical Characters like č ć š đ ž

### DISCUSSION THREAD

**Discussion by unbranded-hr** [dompdf/dompdf#3110] [General]
Hello, I am from Croatia, and we have Diacritical Characters like "Č Ć Š Đ Ž, č ć š đ ž". The PDF shows only Š Ž. For Č Ć Ž it shows a "?" can somebody help please.
![Snimka zaslona 2023-01-13 094853](https://user-images.githubusercontent.com/70441900/212295894-f6dd3f86-3cda-4e8c-a1f8-f3bbf8e9c8b2.jpg)

**Comment by bsweeney** [comment]
The core PDF fonts (Helvetica, Time, Courier, and the generic families sans-serif, serif, and monospace) only support Windows ANSI encoding. You have to style your text with a font that supports Unicode encoding. For these particular characters you can use the bundled Deja Vu fonts.

```
<div style="font-family: DejaVu Sans;">Č Ć Š Đ Ž, č ć š đ ž</div>
```

**Reply by unbranded-hr** [reply]
Thank you very much. YES, it works :-)

**Comment by unbranded-hr** [comment]
Unfortunately it does not work when I download the document, it shows again "?" and not Č Ć Š Đ Ž, č ć š đ ž

$dompdf->stream($fileName, Array('Attachment'=>1));

**Reply by bsweeney** [reply]
Before I could provide more guidance, I would need to see a sample of your HTML+CSS as well as the PHP code you're using to render the PDF.

**Comment by unbranded-hr** [comment]
**For example, I use this to display the Customer's Info (it's a receipt.)**

![Snimka zaslona 2023-01-18 152114](https://user-images.githubusercontent.com/70441900/213195627-42fc278f-153b-4640-8941-3f4bf07f798b.jpg)

![Snimka zaslona 2023-01-18 150943](https://user-images.githubusercontent.com/70441900/213193299-a16ef5c7-de6e-4b00-9468-f1c3e3808e11.jpg)

**Displaying in Chrome looks good, but after downloading it the font changes and the Logo changes too.**

`$dompdf->stream($fileName, Array('Attachment'=>1));
`
![Group 48 (1)](https://user-images.githubusercontent.com/70441900/213194846-d208fe5e-5730-46e1-8e36-d9715fe618c6.png)


--------------------------------------------


![Group 49 (1)](https://user-images.githubusercontent.com/70441900/213194876-d3a0dab8-18a0-461f-8f82-d8c19f8cff8f.png)

**Reply by bsweeney** [reply]
Are you using the download button in Chrome or downloading by some other method?

**Reply by unbranded-hr** [reply]
When I use this button, everything is fine. 

![Snimka zaslona 2023-01-18 195504](https://user-images.githubusercontent.com/70441900/213269739-eb8c8910-5c0c-4873-87fe-19cfd6c79066.jpg)

only if use this code
`$dompdf->stream($fileName, Array('Attachment'=>1)); `
